vespalib/src/vespa/vespalib/util/brain_float16.h

```diff
@@ -0,0 +1,104 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+#include <bit>
+#include <cstdint>
+#include <cstring>
+
+namespace vespalib {
+
+class BrainFloat16 {
```

Comment:
Perhaps name it as the better known BFloat16 as that seems to be the more widely used name.

Reply:
Yes, I agree. I guess the file names should reflect this as well.

Reply:
Please also add class description. Could link to https://en.wikipedia.org/wiki/Bfloat16_floating-point_format as well.

Reply:
fixed.